src/sympc/__init__.py

```diff
@@ -53,3 +53,64 @@ def add_methods_tensor_syft() -> None:
     del get_distribution, DistributionNotFound
 
 add_methods_tensor_syft()
+
+allowed_operations_on_share_tensor = [
```

Comment:
Q: Could you move this in the `tensor` folder and here expose this through the `__all__` variable?